code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -79,14 +79,14 @@
 
 	var/sunder_recov = xeno_caste.sunder_recover * -0.5 //Baseline
 
-	if(resting) //Resting doubles sunder recovery
-		sunder_recov *= 2
+	if(resting) //Resting gives a 50% bonus to sunder recovery.
+		sunder_recov *= 1.5
```

Comment:
tbh i think the resting one could just be removed overall, since i think sunder healing SHOULNDT be par on par with normal healing